### ISSUE TITLE
Add setting the address bar at the bottom of the screen

### ISSUE THREAD

**Issue by Mojas84** [ImMainTheme/ArchyVivaldi#2]
Hi. Would it be possible to use the main address bar at the bottom in your ARC theme? As used here = https://github.com/KiKaraage/FrameUIforFirefox. And please setting for sidebar size. Tabs in sidebar not color accent.....

**Comment by ImMainTheme** [comment]
Hi, sidebar size could be edit in the first line in the css file. 
Tabs in sidebar should be color accent if you not use my theme.
Main address bar at the bottom I might work for it later, that firefox ui look awesome!
Thanks for your request!

**Comment by Mojas84** [comment]
Thanks very much side bar i setting on 248px it is good size. 😉 No i use my theme. Have like much windows themes. Now use GIN24 theme darker color for tabs... Color accent theme css not use but use highlight color. I just want it to have the same colors as the sidebar...  🤔

**Comment by Mojas84** [comment]
Ok FrameUI is good super it using better a faster.... But keep otherwise everything with Archyvivaldi....

**Comment by Mojas84** [comment]
Have Discord?

**Comment by Mojas84** [comment]
![Sunny_20240421_182657](https://github.com/ImMainTheme/ArchyVivaldi/assets/108186899/ee889d31-44b5-4721-b214-b472c502cd11)
This my photo theme tabs on sidebar not color...........

**Comment by ImMainTheme** [comment]
you mean the text or the background of each tab?

**Comment by ImMainTheme** [comment]
I found I wrote the readme wrong when I saw your photo. 
In top bar, the space should be after the download.

**Comment by Mojas84** [comment]
> I found I wrote the readme wrong when I saw your photo.
> In top bar, the space should be after the download.

Yes I thought it was kind of weird.

**Comment by Mojas84** [comment]
> you mean the text or the background of each tab?

Yes of course backround used tab text is good....

**Comment by ImMainTheme** [comment]
Hmmm I checked my css not changing any color of background tab. Maybe it caused by your theme. 
Please check if your theme toggled transparent background tab.

**Comment by Mojas84** [comment]
Yes work it good. My error on vivaldi are not experience with theme create....😆 THX and Sorry for this....

**Comment by Mojas84** [comment]
![Sunny_20240421_200943](https://github.com/ImMainTheme/ArchyVivaldi/assets/108186899/0c173c11-a6a1-4ea1-8d30-dadf962adc45)

**Comment by ImMainTheme** [comment]
Hi, I did updated for address bar on bottom.
Please use the new css to check if it work as expected.
And I suggest to disable show separator below pinned tabs on settings-> tabs -> pinned tabs.
I made my own separator since the original one would disappear if there are only pinned tabs.
I would put this information in readme too.

**Comment by Mojas84** [comment]
> Hi, I did updated for address bar on bottom. Please use the new css to check if it work as expected. And I suggest to disable show separator below pinned tabs on settings-> tabs -> pinned tabs. I made my own separator since the original one would disappear if there are only pinned tabs. I would put this information in readme too.

Woow super work you are best customizer....🤩

**Comment by Mojas84** [comment]
![Sunny_20240421_220141](https://github.com/ImMainTheme/ArchyVivaldi/assets/108186899/e9fdbb34-b35a-4814-becb-32f33cea46e6)
But now after the new setting, my address bar is recolored with the background color and not the highlight..... Can't you set your own color just for the address bar??

**Comment by ImMainTheme** [comment]
ohh that's some code I've forgot to change.
Please go to line 51
delete ".below" on that line.
from 
".UrlBar-AddressField.button-textonly.below:not(:focus-within) {" 
to 
".UrlBar-AddressField.button-textonly:not(:focus-within) {"

or just download the latest css file again

**Comment by Mojas84** [comment]
> ohh that's some code I've forgot to change. Please go to line 51 delete ".below" on that line. from ".UrlBar-AddressField.button-textonly.below:not(:focus-within) {" to ".UrlBar-AddressField.button-textonly:not(:focus-within) {"
> 
> or just download the latest css file again

Download again cool super....👍

**Comment by Mojas84** [comment]
Would it be possible to change the Vivaldi icon in the address bar for settings, etc. to the ARC icon??
![8cxECVs](https://github.com/ImMainTheme/ArchyVivaldi/assets/108186899/85b2770f-571d-4469-ad9f-6563b8cd1f62)

**Comment by ImMainTheme** [comment]
you can add this code to your css.  you can also adjust the size by change those 28px.
```
button.vivaldi {
    background-image: url(https://upload.wikimedia.org/wikipedia/commons/3/37/Arc_%28browser%29_logo.svg);
    background-size: 28px 28px;
    background-repeat: no-repeat;
    background-position: center;
}

span.vivaldi-v {
    display: none;
}
```

**Comment by Mojas84** [comment]
THX Super!!!🙂
